libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.service.ts

```diff
@@ -65,18 +66,33 @@ export class ParentalLeaveService {
     }
   }
 
+  async getAttachments(application: Application): Promise<Attachment[] | null> {
```

Comment:
I find it cleaner to have an empty array instead of null when there are no items. But that is just my preference :)

Reply:
I followed your recommendation, looks better, thanks 👍